calculator/operations.py

```diff
@@ -1,23 +1,43 @@
+from __future__ import division
 
 def add(*args):
-    # your implementation here
-    pass
+        if len(args) == 1:
```

Comment:
I'm sorry, nekatak. My mistake. What i meaned was to use sum() instead of lambda. Something like return sum(args). It should work even when you pass nothing to the function call. It worked for us. Just a suggestion.